synapse/logging/opentracing.py

```diff
@@ -1032,11 +1034,11 @@ def trace_servlet(
             # with JsonResource).
             scope.span.set_operation_name(request.request_metrics.name)
 
-            # set the tags *after* the servlet completes, in case it decided to
-            # prioritise the span (tags will get dropped on unprioritised spans)
             request_tags[
                 SynapseTags.REQUEST_TAG
             ] = request.request_metrics.start_context.tag
```

Comment:
I wish I could just add the tag instead but I don't know how to get access to the `hs` object here,

```py
request_tags[SynapseTags.INSTANCE_NAME] = hs.get_instance_name()
```